OuterInnerVariableRef.java

```diff
@@ -0,0 +1,56 @@
+package com.baidu.bugbye.bca.demo;
+
+public class OuterInnerVariableRef {
+
+    // 增加第二处扫描信息
```

Comment:
此处不通过，修改一下提示信息
```suggestion
    // 测试测试
```

> test
  
Duplicate of #`代码`
#2

Reply:
此处我觉得是正确的，不需要

Reply:
aaa

Reply:
我看看